bool armedStatus	#true if system is armed
